NuGetPackages/MonoGame.Framework.iOS.nuspec

```diff
@@ -0,0 +1,24 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/10/nuspec.xsd">
+    <metadata>
+        <id>MonoGame.Framework.iOS</id>
+        <version>3.3.0</version>
```

Comment:
Solved...  nothing to do here.  TeamCity allows us to overwrite the version from the NuSpec file.  So it will get the same versioning as each build automatically.